CONTRIBUTING.md

```diff
@@ -0,0 +1,586 @@
+			border: 1px solid #faebcc;
+			border-radius: 4px;
+		}
+    </style>
+  </head>
+
+  <body>
+
+**Tilezen** loves contributions from community members like you! Contributions come in many different shapes and sizes. In this file we provide guidance around two of the most common types of contributions: opening issues and opening pull requests.
+
+# Community Values
+
+We ask that you are respectful when contributing to Tilezen or engaging with our community. As a community, we appreciate the fact that contributors might be approaching the project from a different perspective and background. We hope that beginners as well as advanced users will be able to use and contribute back to Tilezen. We want to encourage contributions and feedback from all over the world, which means that English might not be a contributor's native language, and sometimes we may encounter cultural differences. Contructive disagreements can be essential to moving a project forward, but disrespectful language or behavior will not be tolerated.
+
+Above all, be patient, be respectful, and be kind!
+
+# Submitting Issues
+
+Most issues for Tilezen are housed in the [Tilezen/vector-datasource](https://github.com/Tilezen/vector-datasource) repo. Before opening an issue, be sure to search the repository to see if someone else has asked your question before. If not, go ahead and [open a new issue](https://github.com/Tilezen/vector-datasource/issues/new).
+
+## Submitting technical bugs
+
+When submitting bug reports, please be sure to give us as much context as possible so that we can reproduce the error you encountered. Be sure to include:
+
+- System conditons (operating system, browser, etc), if you're running from source
+- Steps to reproduce
+- Expected outcome
+- Actual outcome
+- Screenshots, if applicable
+- Code that exposes the bug, if you have it (such as a failing test or a basic script)
+
+## Submitting issues around vector tile quality
+
+It's important to get feedback about the quality of local tile results. Your local knowledge will make it easier for us to understand the problem. When submitting issues be sure to include:
+
+- Where in the world you were looking (a placename &/or zoom, latitude, longitude)
+- Your tile query
+- Your expected result
+- Your actual result
+
+
+# Pull Requests Welcome!
+
+## Project standards overview
+
+Tilezen has several miscellaneous standards:
+
+- we follow [PEP8](https://www.python.org/dev/peps/pep-0008/) coding style for Python and use [Flake8](http://flake8.pycqa.org/en/latest/) to enforce those conventions
+- we *love* tests, [check them out](https://github.com/tilezen/vector-datasource/tree/master/integration-test)
+- we use [CircleCI](https://circleci.com/gh/mapzen/vector-datasource) for continuous integration testing
+- we use [semver](http://semver.org/) for package versioning
+
+All  unit tests in a project will be automatically invoked when you commit to an existing project; make sure they exit successfully!
```

Comment:
Nitpick on spacing "All[ ][ ] unit".

And it sounds a little more natural (to me) to phrase this as: "make sure they pass".